src/checkbox/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import styles from './styles.css';
+import { Icon } from 'infl-icons';
+import { connectTheme } from 'gild';
+
+import mapPropsToCss from './theme';
+
+const cx = classNames.bind(styles);
+const checkIcon = 'check';
+const indeterminateIcon = 'minus';
+const requiredIcon = 'asterisk';
+
+
+// TODO: change to stateless component when gild is updated
+class Checkbox extends React.Component {
+  render() {
+    const { disabled, onChange, indeterminate, isChecked, label, required, theme } = this.props;
+
+    return (
+      <div
+        onClick={disabled ? null : onChange}
+        className={styles['pt-checkbox']}>
```

Comment:
Since we're binding to styles anyways, let's use cx everywhere. So this would just be:

`cx('pt-checkbox')`